eda/src/autogluon/eda/state.py

```diff
@@ -35,6 +38,8 @@ def __dict__(self):
 
 
 class StateCheckMixin:
+    logger = logging.getLogger(__name__)
+
     def at_least_one_key_must_be_present(self, state: AnalysisState, *keys):
```

Comment:
While we're here, we could add return types to these functions

Reply:
done